Content.Server/Atmos/Components/BarotraumaComponent.cs

```diff
@@ -26,6 +26,22 @@ public sealed class BarotraumaComponent : Component
         ///     These are the inventory slots that are checked for pressure protection. If a slot is missing protection, no protection is applied.
         /// </summary>
         [DataField("protectionSlots")]
-        public List<string> ProtectionSlots = new() { "head", "outerClothing" }; 
+        public List<string> ProtectionSlots = new() { "head", "outerClothing" };
+
+        /// <summary>
+        /// Cached pressure protection values
+        /// </summary>
+        [ViewVariables]
+        public float HighPressureMultiplier = 1f;
+        [ViewVariables]
+        public float HighPressureModifier = 0f;
+        [ViewVariables]
+        public float LowPressureMultiplier = 1f;
+        [ViewVariables]
+        public float LowPressureModifier = 0f;
+
+        [ViewVariables]
+        public bool Immuned = false;
```

Comment:
I think this is fine to still be a TryComp, at a minimum it needs a comment because "Immuned" is pretty vague.